### PR TITLE
API video ssrc change

### PR DESCRIPTION
**Pull request by olofkallander**

### CODE REVIEW

File: api/EndpointDescription.h [hunk 1]
```diff
@@ -91,14 +89,26 @@ struct EndpointDescription
...
     {
-        utils::Optional<Transport> _transport;
-
-        std::vector<uint32_t> _ssrcs;
-        std::vector<SsrcGroup> _ssrcGroups;
-        std::vector<PayloadType> _payloadTypes;
-        std::vector<std::pair<uint32_t, std::string>> _rtpHeaderExtensions;
-        utils::Optional<std::vector<uint32_t>> _ssrcWhitelist;
-        std::vector<SsrcAttribute> _ssrcAttributes;
+        utils::Optional<Transport> transport;
```
Comment (marcusspangenberg) [inline api/EndpointDescription.h:92]: I think you forgot the member prefix _

**Comment by marcusspangenberg** [comment]
Is there a change in the API related to this? Will you update the examples/docs as well?

**Comment by olofkallander** [comment]
> Is there a change in the API related to this? Will you update the examples/docs as well? 

Yes, they should be updated already.

**Review by RicardoMDomingues — APPROVED** [review]
LGTM, @olofkallander I made some fixes. Please take a look

File: api/Parser.cpp [hunk 2]
```diff
@@ -357,53 +357,40 @@ EndpointDescription parsePatchEndpoint(const nlohmann::json& data, const std::st
...
+            videoChannel.streams.emplace_back();
+            auto& videoStream = videoChannel.streams.back();
+            for (const auto& rtpSource : requiredJsonArray(stream, "sources"))
             {
-                const auto ssrc =
-                    ssrcJson.is_string() ? std::stoul(ssrcJson.get<std::string>()) : ssrcJson.get<uint32_t>();
-                ssrcAttribute._ssrcs.push_back(ssrc);
+                api::EndpointDescription::VideoStream::Level level;
+                const auto& main = rtpSource["main"];
+                level.main = (main.is_string() ? std::stoul(main.get<std::string>()) : main.get<uint32_t>());
```
Comment (olofkallander) [inline api/Parser.cpp:385]: Should we really support this. We specify the API and there is no point in doing ssrc as string, and no examples of it.

**Pull request merged**